Core/include/Acts/EventData/VectorMultiTrajectory.hpp

```diff
@@ -420,6 +421,36 @@ class VectorMultiTrajectory final
         &m_measCov[offset]};
   }
 
+  TrackStateProxy::EffectiveMeasurement effectiveMeasurement_impl(
```

Comment:
Do you a solution in mind? I guess we could expose the starting pointer of the measurement. Then the frontend would be able build the maps.

Reply:
I think in the measurement selector I've had a similar problem, and the somewhat hacky solution I used there was to use a wrongly sized measurement dim 1 access and then unpacked the pointer from that. 

That's a bit dangerous, but with a good comment it might be worth the saved effort in the backend. I think this would also make this non-breaking.

Reply:
Ah right I saw this and replaced it with the dynamic version now. I will copy the comment and use the same solution without touching the backend then.

Something else: Do you have an opinion on the naming? I am not too happy with `effectiveMeasurement` / `effectiveCalibrated`. I took this from the projector version but I feel like it does not apply so well here. The name `measurement` won't work because of the clash with the templated version I guess. Maybe `measurementDynamic`?